src/NHSD.BuyingCatalogue.Ordering.Api/Startup.cs

```diff
@@ -21,23 +21,21 @@ public sealed class Startup
     {
         private readonly IWebHostEnvironment _environment;
 
+        private readonly IConfiguration Configuration;
+
         public Startup(IWebHostEnvironment environment, IConfiguration configuration)
         {
             Configuration = configuration;
             _environment = environment;
         }
 
-        public IConfiguration Configuration { get; }
-
-        // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
             var connectionString = Configuration.GetConnectionString("OrderingDb");
-            var authority = Configuration.GetValue<string>("authority");
+            var authority = Configuration.GetValue<string>("IssuerUrl");

```

Comment:
After looking at [OAPIs Startup](https://github.com/nhs-digital-gp-it-futures/BuyingCatalogueIdentity/blob/development/src/NHSD.BuyingCatalogue.Organisations.Api/Startup.cs); for consistency sake, do you think it would be a good idea to rename the property in appsettings.json from IssuerUrl -> authority  (and then obviously revert this to `var authority = Configuration.GetValue<string>("authority");`) ?

Reply:
Yes, tbh I was referencing Identity rather than OAPI. Will make it a capital "A", as from Sams comment from earlier, to make sure they all have the same casing.